concepts/components-events-groups.md

```diff
@@ -64,24 +64,79 @@ An example:
 },
 ```
 
-Like component-groups, events are 100% custom created inside each entity. You cannot use events from other entities in your own. Do not be temted to use `"minecraft:ageable_grow_up"` in your own entity. If you want grow-up functionality, you need to define the component-groups and events yourself.
+Conditional events are events using "filters" to return a component group depending on your filter see the below example.
```

Comment:
Can you please put this below my explanation of events? Its weird to start with an example.